packages/twenty-server/src/engine/api/graphql/graphql-query-runner/helpers/process-aggregate.helper.ts

```diff
@@ -1,10 +1,13 @@
+import { Injectable } from '@nestjs/common';
+
 import { SelectQueryBuilder } from 'typeorm';
 
 import { AGGREGATE_OPERATIONS } from 'src/engine/api/graphql/graphql-query-runner/constants/aggregate-operations.constant';
 import { AggregationField } from 'src/engine/api/graphql/workspace-schema-builder/utils/get-available-aggregations-from-object-fields.util';
 import { formatColumnNamesFromCompositeFieldAndSubfields } from 'src/engine/twenty-orm/utils/format-column-names-from-composite-field-and-subfield.util';
 import { isDefined } from 'src/utils/is-defined';
 
+@Injectable()
```

Comment:
is this useful?

Reply:
It's just to start following the NestJS injection pattern, as we can later need some stuff from injectable class inside this class like `FeatureFlag`